android/src/main/java/vn/luongvo/kmm/survey/android/ui/screens/survey/views/SurveyQuestion.kt

```diff
@@ -84,19 +93,42 @@ fun SurveyQuestion(
     }
 }
 
+@Composable
+private fun AnswerForm(
+    modifier: Modifier = Modifier,
+    question: QuestionUiModel
+) {
+    with(question) {
+        when (displayType) {
+            DisplayType.STAR -> StarRatingBar(
+                answers = answers,
+                onValueChange = {
+                    Timber.d("$displayType -> onInputChange: $it")
+                },
+                modifier = modifier
+            )
+            DisplayType.HEART -> HeartRatingBar(
+                answers = answers,
+                onValueChange = {
+                    Timber.d("$displayType -> onInputChange: $it")
+                },
+                modifier = modifier
+            )
+            else -> Unit
+        }
+    }
+}
+
 @Preview
 @Composable
-fun SurveyQuestionPreview() {
+fun SurveyQuestionPreview(
+    @PreviewParameter(SurveyDetailParameterProvider::class) params: SurveyDetailParameterProvider.Params
```

Comment:
```suggestion
    @PreviewParameter(SurveyDetailParameterProvider::class) 
    params: SurveyDetailParameterProvider.Params
```

Reply:
@doannimble I think this could be skipped as it does not reach the line length config 😛

Reply:
I still see a lot of comments related to annotation. Maybe you guys might need to finalize the annotation convention and update it on the compass: https://kotlinlang.org/docs/coding-conventions.html#annotations

![Screenshot 2023-02-08 at 10 30 51](https://user-images.githubusercontent.com/19943832/217422421-fbfce5d5-4e4a-4e80-b39e-3bded3807ef5.png)